### PR TITLE
Add Simulstart and NoAprilFools

### PR DESCRIPTION
**Pull request by Banane9**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -4327,6 +4327,52 @@
...
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/Banane9/NeosNoAprilFools/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosNoAprilFools/releases/download/v1.0.0/NoAprilFools.dll",
```
Comment (XDelta) [inline manifest.json:4346]: +1

File: manifest.json [hunk 2]
```diff
@@ -4327,6 +4327,52 @@
...
+            "category": "Misc",
+            "website": "https://github.com/Banane9/NeosSimulstart",
+            "sourceLocation": "https://github.com/Banane9/NeosSimulstart",
+            "authors": {
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
```
Comment (XDelta) [inline manifest.json:4365]: Note for later, this is windows only.

File: manifest.json [hunk 3]
```diff
@@ -4327,6 +4327,52 @@
...
+                    "releaseUrl": "https://github.com/Banane9/NeosSimulstart/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosSimulstart/releases/download/v1.0.0/Simulstart.dll",
```
Comment (XDelta) [inline manifest.json:4369]: +1

**Review by XDelta — APPROVED** [review]

File: manifest.json (hunk 2 above)
Comment (Banane9) [reply manifest.json:4392]: As far as I can tell, the only thing that's "windows only" is the Administrator part, but the `Verb` property just gets ignored on other systems.

https://developers.redhat.com/blog/2019/10/29/the-net-process-class-on-linux#processname

**Pull request merged**